inc/checks/class-check-meta-tags.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace runcommand\Doctor\Checks;
+
+use WP_CLI;
+
+/**
+ * Warns when there are greater than %threshold_count% plugins activated.
+ */
+class Check_Meta_Tags extends Plugin {
+
+	// Meta tags reg. ex.
+	private $metatags_expression = "/<meta[^>]+(?:name|property)=\"([^\"]*)\"[^>]+content=\"([^\"]*)\"[^>]*>/";
+
+	public function run() {
+
+		// Fetch homepage resposne.
+		$response = wp_remote_get(site_url());
```

Comment:
Rather than make a remote request, we should instead render and parse the template.